tf2onnx/tfonnx.py

```diff
@@ -1638,6 +1638,24 @@ def where_op(ctx, node, name, args):
     return [node, transpose_node]
 
 
+def round_op7(ctx, node, name, args):
+    dtype = ctx.get_dtype(node.input[0])
+    if dtype in [onnx_pb.TensorProto.INT32, onnx_pb.TensorProto.INT64]:
+        node.type = "Identity"
+        return
+
+    # Workaround: for floating point, TF round mode is half to even, while round half up is used here.
+    # Will switch to onnx Round when it is ready.
+    # The difference now is for numbers like 2.5, 4.5 etc.
+    # In practise, it is supposed to have minor impact.
```

Comment:
if there are model having *.5, we will get result mismatch. shall we avoid this?

Reply:
it will not be a good experience to plan for a fix when we have a case with tons of efforts identifying it's this node causing mismatch.

Reply:
make sense, will close this pr.